instrumentation/spring/spring-boot-autoconfigure/src/testLogbackAppender/resources/logback-test.xml

```diff
@@ -9,7 +9,9 @@
     </encoder>
   </appender>
   <appender name="OpenTelemetry"
-    class="io.opentelemetry.instrumentation.logback.appender.v1_0.OpenTelemetryAppender"/>
+    class="io.opentelemetry.instrumentation.logback.appender.v1_0.OpenTelemetryAppender">
+    <captureCodeAttributes>true</captureCodeAttributes>
```

Comment:
do you want to have `false` to see that system property has priority?

Reply:
The default value to capture code attribute is `false`. So, to test the overridding by a system property, a `true` value has to be set in the Logback XML file. The system property overriding is [here](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/4348a3e258751c69002c530b5495ebe76806b667/instrumentation/spring/spring-boot-autoconfigure/src/testLogbackAppender/java/io/opentelemetry/instrumentation/spring/autoconfigure/instrumentation/logging/LogbackAppenderTest.java#L58), and a comment is present [here](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/4348a3e258751c69002c530b5495ebe76806b667/instrumentation/spring/spring-boot-autoconfigure/src/testLogbackAppender/java/io/opentelemetry/instrumentation/spring/autoconfigure/instrumentation/logging/LogbackAppenderTest.java#L89)  explaining that the code attributes capture has been overriden by a system property, so these attribute should not be present.